packages/components/socialFeed/SocialCard/cards/SocialArticleCard.tsx

```diff
@@ -50,176 +50,183 @@ export const SocialArticleCard: FC<{
   style?: StyleProp<ViewStyle>;
   refetchFeed?: () => Promise<any>;
   isFlagged?: boolean;
-}> = memo(({ post, isPostConsultation, refetchFeed, style, isFlagged }) => {
-  const navigation = useAppNavigation();
-  const [localPost, setLocalPost] = useState<Post>(post);
-  const [viewWidth, setViewWidth] = useState(0);
-  const { width: windowWidth } = useWindowDimensions();
+  disabled?: boolean;
+}> = memo(
+  ({ post, isPostConsultation, refetchFeed, style, isFlagged, disabled }) => {
+    const navigation = useAppNavigation();
+    const [localPost, setLocalPost] = useState<Post>(post);
+    const [viewWidth, setViewWidth] = useState(0);
+    const { width: windowWidth } = useWindowDimensions();
 
-  const articleCardHeight = windowWidth < SOCIAL_FEED_BREAKPOINT_M ? 214 : 254;
-  const thumbnailImageWidth = viewWidth / 3;
-  const borderRadius =
-    windowWidth < RESPONSIVE_BREAKPOINT_S ? 0 : SOCIAl_CARD_BORDER_RADIUS;
+    const articleCardHeight =
+      windowWidth < SOCIAL_FEED_BREAKPOINT_M ? 214 : 254;
+    const thumbnailImageWidth = viewWidth / 3;
+    const borderRadius =
+      windowWidth < RESPONSIVE_BREAKPOINT_S ? 0 : SOCIAl_CARD_BORDER_RADIUS;
 
-  const metadata = zodTryParseJSON(
-    ZodSocialFeedArticleMetadata,
-    localPost.metadata,
-  );
-  const oldMetadata = zodTryParseJSON(
-    ZodSocialFeedPostMetadata,
-    localPost.metadata,
-  );
-  const thumbnailImage =
-    metadata?.thumbnailImage ||
-    // Old articles doesn't have thumbnailImage, but they have a file with a isCoverImage flag
-    oldMetadata?.files?.find((file) => file.isCoverImage);
-  const simplePostMetadata = metadata || oldMetadata;
-  const message = simplePostMetadata?.message;
+    const metadata = zodTryParseJSON(
+      ZodSocialFeedArticleMetadata,
+      localPost.metadata,
+    );
+    const oldMetadata = zodTryParseJSON(
+      ZodSocialFeedPostMetadata,
+      localPost.metadata,
+    );
+    const thumbnailImage =
+      metadata?.thumbnailImage ||
+      // Old articles doesn't have thumbnailImage, but they have a file with a isCoverImage flag
```

Comment:
I would recommend to make a new version of this component too since and we will be able to entirely remove this component when we refacto the feed models (hence removing all those hacks)

We can stop maintaining the old articles components after this PR is merged IMO

Reply:
https://github.com/TERITORI/teritori-dapp/pull/1451/commits/271f701c45bbc8a9b93f63def82a5c44eb13bbdf#diff-2322effa5ea2906ae44dbd3959253b8195a179f73e4233c92f49eeb781a7fe19